### PR TITLE
Adds brayns 2.0.0

### PR DESCRIPTION
**Pull request by NadirRoGue**

### CODE REVIEW

File: var/spack/repos/builtin/packages/brayns/package.py [hunk 1]
```diff
@@ -44,20 +45,20 @@ class Brayns(CMakePackage):
     depends_on('assimp@4.1.0', when='@:1.1.0 +assimp')
     depends_on('bbptestdata', type='test', when='+test')
     depends_on('brion', when='+brion')
-    depends_on('deflect ~deflect-qt', when='+deflect')
-    depends_on('freeimage')
+    depends_on('deflect ~deflect-qt', when='@:1.1.0 +deflect')
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/brayns/package.py:52]: Awesome!

I'm wondering if we could move all 1.x dependencies to a block below current dependencies? That would make purging them once 1.0 is deprecated so much easier :)

Comment (NadirRoGue) [reply var/spack/repos/builtin/packages/brayns/package.py:52]: Good point, done!

**Comment by NadirRoGue** [comment]
Build fails because is picking Brion 3.3.1 which lacks new enum entry. I guess Ill retrigger it tomorrow when Brion 3.3.5 is available already?

**Review by matz-e — APPROVED** [review]

File: var/spack/repos/builtin/packages/brayns/package.py (hunk 1 above)
Comment (matz-e) [reply var/spack/repos/builtin/packages/brayns/package.py:52]: Thanks!

**Pull request merged**